lib/active_force/sobject.rb

```diff
@@ -129,6 +129,14 @@ def self.create! args
       new(args).create!
     end
 
+    def self.update(id, attributes)
+      new(attributes.merge(id: id)).update
```

Comment:
I was digging through the source the other day and happened to notice this method. You can probably use it in place of what's in the constructor currently.
https://github.com/Beyond-Finance/active_force/blob/675aad3f19be1ce0a52e3483b93b01a4032776af/lib/active_force/sobject.rb#L247